### DISCUSSION TITLE
umi4版本下，如何在config.tsx中配置ProLayout的token属性？

### DISCUSSION THREAD

**Discussion by myonly1993** [umijs/umi#9966] [Q&A]
umi4版本下，如何在config.tsx中配置ProLayout的token属性。修改导航栏颜色

layout:{token:{ colorBgAppListIconHover: 'rgba(0,0,0,0.06)',}} 下配置无效

theme:{token:{ colorBgAppListIconHover: 'rgba(0,0,0,0.06)',}} 下配置也无效

antd：{
         theme:{
			token:{ colorBgAppListIconHover: 'rgba(0,0,0,0.06)',}
		}

} 下配置也无效

**Comment by fz6m** [comment]
在 `app.ts` 里 layout 的运行时配置。https://umijs.org/docs/api/runtime-config#layout

**Reply by jiajiaxing1** [reply]
请教一下，umi@3版本支持配置这个token吗，我代码中试了不生效

>

**Reply by fz6m** [reply]
不支持，umi 3 默认的 antd 功能是 antd 4 ，如果你需要 token design ，那是 antd 5 。

**Reply by jiajiaxing1** [reply]
好滴 了解了 谢谢
